tests/scripts/seviri.WV-Upper.unprojected_image.sh

```diff
@@ -3,28 +3,26 @@
-geoips run single_source $GEOIPS_TESTDATA_DIR/test_data_seviri/data/20200404.0800_meteoIO_tc2020sh24irondro/H-000-MSG1__-MSG1_IODC___-_________-EPI______-202004040800-__ \
-             $GEOIPS_TESTDATA_DIR/test_data_seviri/data/20200404.0800_meteoIO_tc2020sh24irondro/H-000-MSG1__-MSG1_IODC___-_________-PRO______-202004040800-__ \
-             $GEOIPS_TESTDATA_DIR/test_data_seviri/data/20200404.0800_meteoIO_tc2020sh24irondro/H-000-MSG1__-MSG1_IODC___-WV_062___-000001___-202004040800-C_ \
-             $GEOIPS_TESTDATA_DIR/test_data_seviri/data/20200404.0800_meteoIO_tc2020sh24irondro/H-000-MSG1__-MSG1_IODC___-WV_062___-000002___-202004040800-C_ \
-             $GEOIPS_TESTDATA_DIR/test_data_seviri/data/20200404.0800_meteoIO_tc2020sh24irondro/H-000-MSG1__-MSG1_IODC___-WV_062___-000003___-202004040800-C_ \
-             $GEOIPS_TESTDATA_DIR/test_data_seviri/data/20200404.0800_meteoIO_tc2020sh24irondro/H-000-MSG1__-MSG1_IODC___-WV_062___-000004___-202004040800-C_ \
-             $GEOIPS_TESTDATA_DIR/test_data_seviri/data/20200404.0800_meteoIO_tc2020sh24irondro/H-000-MSG1__-MSG1_IODC___-WV_062___-000005___-202004040800-C_ \
-             $GEOIPS_TESTDATA_DIR/test_data_seviri/data/20200404.0800_meteoIO_tc2020sh24irondro/H-000-MSG1__-MSG1_IODC___-WV_062___-000006___-202004040800-C_ \
-             $GEOIPS_TESTDATA_DIR/test_data_seviri/data/20200404.0800_meteoIO_tc2020sh24irondro/H-000-MSG1__-MSG1_IODC___-WV_062___-000007___-202004040800-C_ \
-             $GEOIPS_TESTDATA_DIR/test_data_seviri/data/20200404.0800_meteoIO_tc2020sh24irondro/H-000-MSG1__-MSG1_IODC___-WV_062___-000008___-202004040800-C_ \
+geoips run single_source $GEOIPS_TESTDATA_DIR/test_data_seviri/data/20231211/0800/H-000-MSG2__-MSG2_IODC___-_________-EPI______-202312110800-__ \
+             $GEOIPS_TESTDATA_DIR/test_data_seviri/data/20231211/0800/H-000-MSG2__-MSG2_IODC___-_________-PRO______-202312110800-__ \
+             $GEOIPS_TESTDATA_DIR/test_data_seviri/data/20231211/0800/H-000-MSG2__-MSG2_IODC___-WV_062___-000001___-202312110800-C_ \
+             $GEOIPS_TESTDATA_DIR/test_data_seviri/data/20231211/0800/H-000-MSG2__-MSG2_IODC___-WV_062___-000002___-202312110800-C_ \
+             $GEOIPS_TESTDATA_DIR/test_data_seviri/data/20231211/0800/H-000-MSG2__-MSG2_IODC___-WV_062___-000003___-202312110800-C_ \
+             $GEOIPS_TESTDATA_DIR/test_data_seviri/data/20231211/0800/H-000-MSG2__-MSG2_IODC___-WV_062___-000004___-202312110800-C_ \
+             $GEOIPS_TESTDATA_DIR/test_data_seviri/data/20231211/0800/H-000-MSG2__-MSG2_IODC___-WV_062___-000005___-202312110800-C_ \
+             $GEOIPS_TESTDATA_DIR/test_data_seviri/data/20231211/0800/H-000-MSG2__-MSG2_IODC___-WV_062___-000006___-202312110800-C_ \
+             $GEOIPS_TESTDATA_DIR/test_data_seviri/data/20231211/0800/H-000-MSG2__-MSG2_IODC___-WV_062___-000007___-202312110800-C_ \
+             $GEOIPS_TESTDATA_DIR/test_data_seviri/data/20231211/0800/H-000-MSG2__-MSG2_IODC___-WV_062___-000008___-202312110800-C_ \
```

Comment:
Here is a link to the data. Hosted on next cloud now.

Reply:
https://io2.cira.colostate.edu/s/XxP4r8MSPesExQr

Reply:
Maybe we can create a small PR to add that integration test script. If we want to wait on this PR, we can probably get some data from Jason Apke or another folk at CIRA / NRL that have routines that take in multiple time steps. I'm not familiar with that sort of processing and would feel more comfortable getting the data from someone that has an expected output.

Reply:
@evrose54 if you want to create an issue for this and link it here, that's fine

Reply:
I added an integration test for this. See modified ``full_install.sh`` and ``full_test.sh``.

Reply:
Thanks!